Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.5.3
```

Comment:
I'm planning a base image for testing here, but it is not finished yet: https://hub.docker.com/r/ruflin/beat-testing/